src/main/java/frc/robot/subsystems/ShooterTest.java

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems;
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import com.ctre.phoenix6.controls.VelocityVoltage;
+import com.ctre.phoenix6.hardware.TalonFX;
+
+public class ShooterTest extends SubsystemBase {
+  /** Creates a new ShooterTest. */
+  private double shooterSpeed = 0.0;
+  private VelocityVoltage velocity = new VelocityVoltage(shooterSpeed);
+  private TalonFX happyMotor2 = new TalonFX(0);
+
+  public ShooterTest() {}
```

Comment:
Nit: Default constructor is the only constructor and it's empty so you don't have to specify it.